packages/webpack-config-single-spa/lib/webpack-config-single-spa.js

```diff
@@ -43,8 +42,9 @@ function webpackConfigSingleSpa(opts) {
       filename: `${opts.orgName}-${opts.projectName}.js`,
       libraryTarget: "system",
       path: path.resolve(process.cwd(), "dist"),
-      jsonpFunction: `webpackJsonp_${opts.projectName}`,
+      // jsonpFunction: `webpackJsonp_${opts.projectName}`,
       devtoolNamespace: `${opts.projectName}`,
+      publicPath: "",
```

Comment:
webpack@5 has an "auto" public path option. unfortunately it doesn't work with loading via systemjs right now. Since `"auto"` is the new default for `publicPath`, I had to specify the public path as an empty string so that the public path comes from systemjs-webpack-interop instead of the broken `"auto"` functionality

Reply:
What makes `auto` incompatible with systemJs?

Reply:
didn't look into it